ONSdigital/eq-survey-runner
# eq-861 Answer Label Incorrect Semantics

### What is the context of this PR?

Fixes #872 by replacing the semantically incorrect `label` element with a `legend`.

- Changed logic in question template so that, if a fieldset/legend is used the legend is hidden rather than replacing the `h2` on the question title.
- Added `field__legend` styling for new legend element.
- Legend falls back to repeating question title if no "answer label" is supplied. Also visually hidden in that case.
- Improved semantics of UKIS survey by moving "Select all that apply" out of answer label (and therefore legend) and into question description wherever possible.

### How to review 

- start UKIS survey
- Check that semantics are correct for as many questions as possible. IE. a `legend` is used where there are fields with multiple `input` (ie radios/checkboxes)
